src/cljs/rems/user_settings.cljs

```diff
@@ -80,10 +80,22 @@
                    (when first-time? ; to avoid an infinite loop if the settings fail to save (e.g. unsupported language in cookies)
                      [[:check-if-should-save-language!]]))})))
 
+(rf/reg-event-fx
+ :loaded-user-settings-fail
+ (fn [{:keys [db]} [_ user-settings-error]]
+   (let [logged-in? (:user (:identity db))]
+     (if-not logged-in?
+       (if-not (= 401 (:status user-settings-error))
+         {:db db
+          :dispatch (flash-message/show-default-error! :top (str "Fetch user settings"))}
+         {:db db})
+       {:db db}))))
+
 (defn fetch-user-settings! []
   (fetch "/api/user-settings"
-         {:handler #(rf/dispatch-sync [:loaded-user-settings %])
-          :error-handler (flash-message/default-error-handler :top "Fetch user settings")}))
+         {:simple-error-handler? true
```

Comment:
I'm not sure why this was renamed to `:simple-error-handler?` but I think it's backwards. Isn't the normal i.e. simple case that you wrap the default error handler? And this would then be the ... `:wrap-default-error-handler? false` or even `:custom-error-handler true` case. Do you see what I mean?

Reply:
same here, need to go through the ticket again to think of the better naming

Reply:
Do you think naming is more compelling now?

Reply:
Actually I don't. I mean if I am passing `:error-handler XXX` to me it means I'm also passing a custom error handler so I don't understand why I need to pass another parameter called `:custom-error-handler` to do something else? It affects the wrapping so I would've liked to see that word in it. :shrug: